code/game/objects/structures/rocks.dm

```diff
@@ -59,6 +126,18 @@
 	desc = "A pile of ancient fossils. There are some oddly shaped skulls in here..."
 	icon_state = "lavarocks3"
 
+/obj/structure/rock/tinyrock
+	name = "tiny rock pile"
+	desc = "A pile of tiny pebbles..."
+	icon_state = "tinyrock_3"
+	density = FALSE
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	max_integrity = 100
+
+/obj/structure/rock/tinyrock/Initialize(mapload)
+	. = ..()
+	icon_state = "tinyrock_[rand(1,8)]"
```

Comment:
there is a /variable subtype of rock that has icon_state variation built in, better to be consistant and use that.